src/lib/DisposableEventListener.ts

```diff
@@ -1,5 +1,9 @@
-import Disposable from 'seng-disposable';
-import { EventHandler } from 'seng-event/lib/EventDispatcher';
+export type Listener = EventListenerOrEventListenerObject;
+
+export interface EventDispatcher {
+  addEventListener(type: string, listener: Listener, useCapture?: boolean): void;
+  removeEventListener(type: string, listener: Listener, useCapture?: boolean): void;
```

Comment:
These typings are wrong/old:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
And of course, so is our implementation.

Reply:
good point

Reply:
Types are updated. Please review again.